### PR TITLE
Update docs in Bamboo.Phoenix

### PR DESCRIPTION
**Pull request by paulcsmith**
After looking through this I think "template" might be the more correct word here

### CODE REVIEW

File: lib/bamboo/phoenix.ex [hunk 1]
```diff
@@ -15,10 +15,10 @@ defmodule Bamboo.Phoenix do
         def text_and_html_email_with_layout do
           new_email()
           # You could set just a text layout or just an html layout
-          |> put_text_layout
-          |> put_html_layout
+          |> put_text_layout({MyApp.LayoutView, "email.text"})
+          |> put_html_layout({MyApp.LayoutView, "email.html"})
```
Comment (paulcsmith) [inline lib/bamboo/phoenix.ex]: I think adding the arguments might make things clearer

File: lib/bamboo/phoenix.ex [hunk 2]
```diff
@@ -122,7 +122,7 @@ defmodule Bamboo.Phoenix do
 
       def text_email_layout do
         new_email
-        # Will use MyApp.LayoutView with email.text layout when rendering text emails
+        # Will use MyApp.LayoutView with email.text template when rendering text emails
```
Comment (paulcsmith) [inline lib/bamboo/phoenix.ex]: I'm not sure about this, but I think template is probably more correct. Thoughts?

Comment (anellis) [inline lib/bamboo/phoenix.ex]: Hmm I keep going back and forth on this one. In the method definitions though we have
`put_text_layout(email, layout)`, so right now in a variety of places we're referring to that `email.text` or `email.html` as a layout. So, idk. I'd be fine with referring to it as a template but if so I think we need to change it everywhere.

Comment (anellis) [inline lib/bamboo/phoenix.ex]: Actually, sorry, just saw that its actually broken down to `{layout, template}`, so yes, 👍  on the change

**Comment by anellis** [comment]
LGTM

**Comment by paulcsmith** [comment]
Thanks for the review!

**Pull request merged**